includes/class-freemius.php

```diff
@@ -5855,14 +5941,20 @@ function get_module_label( $lowercase = false ) {
 		 * @author Vova Feldman (@svovaf)
 		 * @since  1.0.4
 		 *
+         * @param string $plugin_main_file_path
+         *
 		 * @return string
 		 */
-		function get_plugin_basename() {
+		function get_plugin_basename( $plugin_main_file_path = false ) {

```

Comment:
@fajardoleo you've added `$plugin_main_file_path` as an argument, but I don't see any call to this method with an argument.